batch/interface.go

```diff
@@ -25,6 +25,22 @@ type Request struct {
 	//
 	// NOTE: This field is optional.
 	OnCommit func(commitErr error) error
+
+	// force should be true if we immediately want to execute this request
+	// when it comes in.
+	force bool
+}
+
+// SchedulerOption is a type that can be can supply options to a scheduled
+// request.
+type SchedulerOption func(r *Request)
+
+// PriorityAdd will make the request be executed immediately, skipping the
+// batch interval.
+func PriorityAdd() SchedulerOption {
```

Comment:
Could turn this on its head, and make priority add the default. Then remote updates could be given a `LazyAdd` option to indicate they can be batched.

Reply:
made lazy the option